src/test/scala/com/microsoft/hyperspace/index/HybridScanGeneralTest.scala

```diff
@@ -0,0 +1,167 @@
+/*
+ * Copyright (2020) The Hyperspace Project Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.microsoft.hyperspace.index
+
+import org.apache.hadoop.fs.Path
+import org.apache.spark.sql.DataFrame
+import org.apache.spark.sql.execution.datasources._
+
+import com.microsoft.hyperspace.Hyperspace
+
+class HybridScanGeneralTest extends HybridScanTestSuite {
+  override val systemPath = new Path("src/test/resources/hybridScanTest")
+
+  override def beforeAll(): Unit = {
+    super.beforeAll()
+    import spark.implicits._
+    val dfFromSample = sampleData.toDF("Date", "RGUID", "Query", "imprs", "clicks")
+    hyperspace = new Hyperspace(spark)
+
+    dfFromSample.write.format(fileFormat).save(sampleDataFormatAppend)
+    dfFromSample.write.format(fileFormat).save(sampleDataFormatAppend2)
+    dfFromSample.write.format(fileFormat).save(sampleDataFormatDelete)
+    dfFromSample.write.format(fileFormat).save(sampleDataFormatDelete2)
+    dfFromSample.write.format(fileFormat).save(sampleDataFormatDelete3)
+    dfFromSample.write.format(fileFormat).save(sampleDataFormatDelete4)
+    dfFromSample.write.format(fileFormat).save(sampleDataFormatBoth)
+    dfFromSample.write.format(fileFormat2).save(sampleDataFormat2Append)
+    dfFromSample.write.format(fileFormat2).save(sampleDataFormat2Delete)
```

Comment:
Can we remove unused sample data?

Reply:
They are all used.. do you think it's not necessary to run the testset for both partitioned data and non-partitioend data?

Reply:
Ah sorry, I thought this suite was testing only the tests where partitioned data suite couldn't test.